backend/models.py

```
@@ -200,10 +216,22 @@ class UserModule(db.Model):
         db.Boolean, default=False
     )  # flag for if the user can start the module
     completed_date = db.Column(db.DateTime, nullable=True)  # used to help with review
+    submitted_runtime = db.Column(db.String, nullable=True)  # only for challenges
 
     user = db.relationship("User", back_populates="modules")
     module = db.relationship("Module", back_populates="users")
 
+    @validates("submitted_runtime")
+    def validate_submitted_runtime(self, key, value):
```

Ruff (0.8.2)

Missing return type annotation for public function `validate_submitted_runtime`

(ANN201)